explorer/client/src/components/events/eventDisplay.tsx

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    isMoveEvent,
```

Comment:
As an aside, I wonder if there should be an API to just get the type of event, and then an event enum that can be switched on.

Reply:
yes, the data format could be friendlier / require less frontend parsing.